data_models/bioinformatics_mill_models/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+from django.utils import timezone
+
+
+class TimeTrackedModel(models.Model):
+    created_at = models.DateTimeField(editable=False)
+    updated_at = models.DateTimeField()
+
+    def save(self, *args, **kwargs):
```

Comment:
I think you won't need this code here if you add the ``auto_now__add`` and ``auto_now`` arguments to ``created_at`` and ``updated_at``, respectively, right? https://docs.djangoproject.com/en/1.10/ref/models/fields/#django.db.models.DateField  I think they will do that automatically for you.